client.rb

```diff
@@ -26,7 +27,8 @@ def initialize
         $logger.fatal "Config: #{CONFIG_FILE} not found!"
         exit
       end
-      @config = YAML.load_file(File.join(__dir__, CONFIG_FILE))
+      @config1 = ERB.new File.new(File.join(__dir__, CONFIG_FILE)).read
```

Comment:
Please name the variable `erb` (here and the line below) instead of `@config1` as it's more meaningful and there's no reason for making it an instance variable.

Reply:
Done in f40abe7